reference-tests/src/test/java/net/consensys/linea/GeneralStateReferenceTestTools.java

```diff
@@ -137,7 +137,17 @@ public static Collection<Object[]> generateTestParametersForConfig(final String[
 
   @SneakyThrows
   public static void executeTest(final GeneralStateTestCaseEipSpec spec) {
-    final BlockHeader blockHeader = spec.getBlockHeader();
+    final BlockHeader blockHeader =
+        BlockHeaderBuilder.createDefault()
+            .baseFee(Wei.of(LINEA_BASE_FEE))
+            .gasLimit(LINEA_BLOCK_GAS_LIMIT)
+            .difficulty(Difficulty.of(LINEA_DIFFICULTY))
+            .number(spec.getBlockHeader().getNumber())
+            .coinbase(spec.getBlockHeader().getCoinbase())
+            .timestamp(spec.getBlockHeader().getTimestamp())
+            .parentHash(spec.getBlockHeader().getParentHash())
+            .buildBlockHeader();
```

Comment:
Is the idea that you are post-hoc modifying the `baseFee`, `gasLimit` and `difficulty` parameters of the block header to reflect expected values of the arithmetization ?

Reply:
Removed this part, as I am using the new zkevm binary for reference tests.